<launch>
  <arg name="manager" default="realsense_ros_camera_manager"/>
  <node pkg="nodelet" type="nodelet" name="$(arg manager)"  args="manager" output="screen"/>

  <arg name="bag_path" default="$(env HOME)/test.bag" />
  <node pkg="rosbag" type="play" name="rosbag" args="$(arg bag_path)"/>
  
  <!-- If there are compressed streams, decompress them during playback -->
  <node name="decompress_color" pkg="image_transport" type="republish" output="screen" args="compressed in:=/camera/color/image_raw raw out:=/camera/color/image_raw" />
  <node name="decompress_depth" pkg="image_transport" type="republish" output="screen" args="compressedDepth in:=/camera/depth/image_raw raw out:=/camera/depth/image_raw" />
</launch>
